src/cpp/session/modules/SessionPackages.R

```diff
@@ -962,29 +962,59 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
    ## Create a DESCRIPTION file
    
    # Fill some bits based on devtools options if they're available.
-   # Protect against vectors with length > 1
-   getDevtoolsOption <- function(optionName, default, collapse = " ")
+   # Protect against vectors with length > 1.
+   getDevtoolsOption <- function(optionName,
+                                 default = NULL,
+                                 collapse = " ")
    {
-      devtoolsDesc <- getOption("devtools.desc")
-      if (!length(devtoolsDesc))
-         return(default)
-      
-      option <- devtoolsDesc[[optionName]]
-      if (is.null(option))
-         return(default)
+      for (descKey in c("usethis.description", "devtools.desc"))
```

Comment:
FWIW usethis dropped support for `devtools.desc` in v2.0.0, released 2020-12-10. Especially given that this is interactive workflow functionality, it seems potentially safe to do so here as well. So, call this `getUsethisOption` and only look in `usethis.description`? But I also doubt that this matters a whole lot in practice.

https://github.com/r-lib/usethis/blob/a225a21f3e336394051584411efb4d351aa9d78a/NEWS.md?plain=1#L627

Reply:
I saw that too, but I figure I'd keep it around just for backwards compatibility (for those of us out there with very old `.Rprofile`s)